osu.Game/Overlays/Profile/Sections/ProfileSubsection.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Rulesets;
+using osu.Game.Users;
+using JetBrains.Annotations;
+
+namespace osu.Game.Overlays.Profile.Sections
+{
+    public abstract class ProfileSubsection : FillFlowContainer
+    {
+        protected readonly Bindable<User> User = new Bindable<User>();
+
+        protected RulesetStore Rulesets { get; private set; }
+
+        protected OsuSpriteText Missing { get; private set; }
```

Comment:
is there a reason why this is exposed in the base as protected? can't this just live as private in `PaginatedProfileSubsection`? this class does nothing with this sprite text.

if this is moved back there, the `missingText` constructor parameter should be, as well.